spec/lib/whitehall_importer/integrity_checker/change_history_check_spec.rb

```diff
@@ -0,0 +1,138 @@
+RSpec.describe WhitehallImporter::IntegrityChecker::ChangeHistoryCheck do
+  let(:published_edition) {
```

Comment:
A small one, when we've got a block spanning more then one line, the convention is to use `do...end` rather than curly braces.